src/utils/utils.ts

```diff
@@ -108,7 +108,16 @@ export const generateItemFromFilename = async (options: {
     const { size } = stats;
 
     // upload file
-    const uploadFilePath = await uploadFile({ mimetype, filepath });
+    let uploadFilePath = null;
+    try {
+      uploadFilePath = await uploadFile({ mimetype, filepath });
+    } catch (e) {
+      // ignore file upload error
+      // continue the process despite one file failure
+      // the file might be empty and we want to ignore the file
+      // todo: send sentry error if unexpected error
+      return null;
```

Comment:
How is this null handled outside of the function? Would it make sense to simply let it throw here and explicitly handle the failure outside?